index.html

```diff
@@ -0,0 +1,218 @@
+                                alt="Medium" /></a></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <section class="container">
+        <div class="container-block">
+            <h2>My Recent Works</h2>
+            <section>
+                <img src="./Images/Placeholder.svg" alt="Placeholder">
+                <h2>Multi-Post Stories</h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.
+                </p>
+                <div class="box-bot">
+                    <div>CSS</div>
+                    <div>html</div>
+                    <div>bootstrap</div>
+                    <div>Ruby</div>
+                </div>
+                <button class="btn btn-style" type="button">See Project</button>
+            </section>
+            <div class="container-full">
+                <div class="container-poppup">
+                    <div class="pupopp">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
+                        </div>
+                    </div>
+                    <button class="btn btn-style" type="button">See Project</button>
+                </div>
+                <div class="container-poppup">
+                    <div class="pupopp">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
+                        </div>
+                    </div>
+                    <button class="btn btn-style" type="button">See Project</button>
+                </div>
+                <div class="container-poppup">
+                    <div class="pupopp">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
+                        </div>
```

Comment:
As requested, the list of technologies should be enclosed in `ul` tags and each technology should be a separate `li` tag. Make this change for all your projects with this section.